packages/common/src/types/scheduler.ts

```diff
@@ -234,3 +236,10 @@ export type ApiCsvUrlResponse = {
         status: string;
     };
 };
+
+export type CompileProjectPayload = {
+    userUuid: string;
```

Comment:
Is this the user that triggered the payload? Could we give it a name like `createdByUserUuid` if so

Reply:
Done